README.md

```diff
@@ -237,6 +237,7 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 - ![v23] [vite-plugin-icons](https://github.com/antfu/vite-plugin-icons) - Access thousands of icons as Vue components.
 - ![v3] [vite-plugin-vuedoc](https://github.com/JasKang/vite-plugin-vuedoc) - Markdown Code block as Vue Preview components.
 - ![v3] [vite-svg-loader](https://github.com/jpkleemans/vite-svg-loader) - Load SVG files as Vue components.
+- [vite-plugin-csv](https://github.com/SirwanAfifi/vite-plugin-csv) Import CSV files in your vite projects
```

Comment:
what kind of support vue? 2/3 or only 3?

Reply:
@Scrum Opps, I should have moved it to `Framework-agnostic Plugins` section, I have just fixed and pushed.